draft-ietf-dhc-rfc8415bis.xml

```diff
@@ -512,7 +512,7 @@
      The client uses a link-local source address or
```

Comment:
In most cases link-local, with one exception: ADDR-REG-INFORM, which is sent from the global address being registered. See Section 4.2 of RFC9686:

> Unlike other types of messages, which are sent from the link-local address of the client, the ADDR-REG-INFORM message MUST be sent from the address being registered.

Might be useful to point this out.